hack/gofmt.sh

```diff
@@ -14,9 +14,9 @@ if [ "${IS_CONTAINER}" != "false" ]; then
 else
   "${CONTAINER_RUNTIME}" run --rm \
     --env IS_CONTAINER=TRUE \
-    --volume "${PWD}:/go/src/github.com/metal3-io/cluster-api-provider-metal3:ro,z" \
+    --volume "${PWD}:/workdir:ro,z" \
     --entrypoint sh \
-    --workdir /go/src/github.com/metal3-io/cluster-api-provider-metal3 \
+    --workdir /workdir \
```

Comment:
maybe call it ipam? 
```suggestion
    --workdir /ipam \
```

Reply:
I am not sure that would work, let me try it locally

Reply:
you need to change the mount path as well